.github/workflows/yarn.yaml

```diff
@@ -0,0 +1,14 @@
+name: Check with yarn
```

Comment:
I would move this to a separate PR and extend it to test multiple package managers via GitHub Action's matrix, if that's the purpose of this workflow.

That would allow the PR to contain only atomic change and be more likely merged in.

Reply:
Can certainly do that if required; but in this case I felt this was necessary, since it seems to be the way to check if it's working with the alternate package manager. The existing Circle CI only runs the tests with npm, and didn't catch the error before.

Reply:
Interesting. Our CircleCI runs failed on npm, yarn and pnpm equally.

Reply:
You mean with #13 or after #13 was merged. It apparently failed also if package-lock.json was not present?
Anyway, I think there's no real reason why there were no GitHub actions here other than this module needing some love from the community. After all, it didn't even have a package-lock.json until #13